specification/cognitiveservices/OpenAI.Inference/package.json

```diff
@@ -4,15 +4,16 @@
   "description": "Azure OpenAI APIs for completions and search",
   "license": "MIT",
   "dependencies": {
-    "@cadl-lang/compiler": "^0.39.0",
-    "@cadl-lang/rest": "^0.39.0",
-    "@cadl-lang/versioning": "^0.39.0",
-    "@cadl-lang/prettier-plugin-cadl": "^0.39",
-    "@azure-tools/cadl-azure-core": "^0.25.0",
-    "@azure-tools/cadl-autorest": "^0.25.0",
-    "@azure-tools/cadl-apiview": "~0.3.1",
-    "@azure-tools/cadl-dpg": "0.25.0",
-    "@azure-tools/cadl-csharp": "0.1.11-beta.20230201.2",
+    "@typespec/compiler": "latest",
+    "@typespec/rest": "latest",
+    "@typespec/http": "latest",
+    "@typespec/versioning": "latest",
+    "@typespec/prettier-plugin-typespec": "latest",
+    "@azure-tools/typespec-azure-core": "latest",
+    "@azure-tools/typespec-autorest": "latest",
```

Comment:
Lets try to keep this set of dependencies minimal to only have the autorest and apiview emitter listed.

Reply:
Hello @weshaggard This PR is for typespec renaming, we will refine the dependency later.